### PR TITLE
Backport ipv6 lane to 0.26

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

**Comment by qinqon** [comment]
ipv6 run 

https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3138/pull-kubevirt-e2e-kind-k8s-1.17.0-ipv6/1238068302236880896

File: github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits-0.26.yaml [hunk 1]
```diff
@@ -454,6 +454,55 @@ presubmits:
           path: /dev/vfio/
           type: Directory
         name: vfio
+  - name: pull-kubevirt-e2e-kind-k8s-1.17.0-ipv6
+    skip_branches:
+    - release-0.26
```
Comment (RamLavi) [inline github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits-0.26.yaml]: doesn't it mean that it will skip release-0.26 branch?

Comment (qinqon) [reply github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits-0.26.yaml]: Bad copy paste s/skip_branches/branches/

**Pull request merged**